.github/workflows/tests.yaml

```diff
@@ -118,19 +114,6 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           slug: ${{ env.MAIN_REPO }}
 
-      - name: Run heavy Pytest (allowed to fail)
```

Comment:
Out of curiosity was there a particular reason for this to be in both `tests` and `heavy-tests` or was it just accidentally duplicated?